html/email_style.css

```
@@ -0,0 +1,437 @@
+        linear-gradient(
+            0deg,
+            rgba(255, 255, 255, 0.012) 1px,
+            transparent 1px
+        );
+    background-size: 40px 40px;
+
+    font-family:
+        "Inter",
+        "HarmonyOS Sans",
+        Arial,
+        sans-serif;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)